### ISSUE TITLE
Article: no repeats format need to fix.

### ISSUE THREAD

**Issue by dwd2010** [FreeCodeCamp/wiki#641]
no repeats format need to fix.
[https://github.com/FreeCodeCamp/wiki/blob/master/Algorithm-No-Repeats-Please.md](https://github.com/FreeCodeCamp/wiki/blob/master/Algorithm-No-Repeats-Please.md)

that format need to fix because this [https://github.com/FreeCodeCamp/camperbot/issues/72](https://github.com/FreeCodeCamp/camperbot/issues/72). so you will see a picture of output from bot at gitter,

**Comment by alayek** [comment]
@abhisekp @SaintPeter I think I can raise a PR to fix the wiki so that @camperbot fetches properly formatted stuff.

What do I have to do? Where do I find the guideline for bot-friendly wiki article?

**Comment by SaintPeter** [comment]
@alayek While we have discussed the camperbot issue, there is no question that this article is not formatted properly.  This needs to be fixed even if we did fix Camperbot. 

See the linked issue (FreeCodeCamp/camperbot#72) for discussion on the Camperbot.  If you want to propose some alternative to the way the Camperbot fetches/displays Wiki articles, we're always open to suggetion.  I tihnk it does "until the first `h2` or XX words" (I don't know how many words exactly.)
